doc/import.md

```diff
@@ -110,6 +111,10 @@ You can run the import with: `bundle exec rails sac_imports:1_people`
 
 ### `sac_imports:nav2-1_roles`
 
+1. Import Mitglied Stammsektion (
+1. Import Zusatzsektion
+1. Import Funktionär
+
```

Comment:
Verstehe nicht ganz was das heisst. Einfach die Reihenfolge in der die Rollen importiert werden?

Reply:
eh ja ... das ist noch leftover notizen ... lösche ich gleich